src/components/timecard/edit-shift-timecard/EditShiftTimecard.jsx

```diff
@@ -1,18 +1,18 @@
+import { PropTypes } from 'prop-types';
 import { useState } from 'react';
 import { Link } from 'react-router-dom';
-import { useTimecardContext } from '../../../context/TimecardContext';
 import EditShiftHours from '../edit-shift-hours/EditShiftHours';
 
-const EditShiftTimecard = () => {
+const EditShiftTimecard = ({ timeEntry, index }) => {
```

Comment:
From the perspective of `editShiftTimecard `I think the variable name `index` isn't too obvious for what the index is referring to (which seems to be the index of the supplied timeEntry within all timeEntries)

Reply:
I've changed the index to a more suitable name